manager/controllers/app/fybrikapplication_controller_test.go

```diff
@@ -62,6 +68,9 @@ var _ = Describe("FybrikApplication Controller", func() {
 			}
 		})
 		It("Test end-to-end for FybrikApplication", func() {
+			if os.Getenv("USE_EXISTING_CONTROLLER") == "true" {
```

Comment:
You changed the environment variable of this condition. With this comparison this test would always be skipped. (Also for the current integration tests)